FeedStoreChallenge/CoreData/Models/ManagedCache.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Daniel Gallego Peralta on 17/8/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
+	@NSManaged var timestamp: Date
+	@NSManaged var feed: NSOrderedSet
+
+	var localFeed: [LocalFeedImage] {
+		return feed.compactMap {
+			($0 as? ManagedFeedImage)?.local
+		}
+	}
+
+	static func find(in context: NSManagedObjectContext) throws -> ManagedCache? {
+		let request = NSFetchRequest<ManagedCache>(entityName: ManagedCache.entity().name!)
+		request.returnsObjectsAsFaults = false
+		return try context.fetch(request).first
+	}
+
+	internal static func newUniqueInstance(in context: NSManagedObjectContext) throws -> ManagedCache {
```

Comment:
`internal` is the default access level, so you don't need to write it explicitly.

It was written explicitly in the first lectures of the program for teaching purposes, but you don't need to do it in your code.

Please remove `internal` modifier

```suggestion
	static func newUniqueInstance(in context: NSManagedObjectContext) throws -> ManagedCache {
```